awx/ui_next/src/components/Lookup/Lookup.jsx

```diff
@@ -371,25 +168,33 @@ const Item = shape({
 
 Lookup.propTypes = {
   id: string,
-  getItems: func.isRequired,
-  lookupHeader: string,
-  name: string,
-  onLookupSave: func.isRequired,
+  header: string,
+  onChange: func.isRequired,
   value: oneOfType([Item, arrayOf(Item)]),
-  sortedColumnKey: string.isRequired,
   multiple: bool,
   required: bool,
-  qsNamespace: string,
+  onBlur: func,
+  qsConfig: QSConfig.isRequired,
+  renderItemChip: func,
+  renderOptionsList: func.isRequired,
 };
 
 Lookup.defaultProps = {
   id: 'lookup-search',
-  lookupHeader: null,
-  name: null,
+  header: null,
   value: null,
   multiple: false,
   required: false,
-  qsNamespace: 'lookup',
+  onBlur: () => {},
+  renderItemChip: ({ item, removeItem, canDelete }) => (
```

Comment:
I can't seem to get the JT add form to load.  It's throwing an error in here about `item.id` not being defined.  See these screenshots: